Src/Base/AMReX_BLProfiler.H

```diff
@@ -484,10 +484,17 @@ inline std::string BLProfiler::CommStats::CFTToString(CommFuncType cft) {
 #define BL_PROFILE_S(fname)
 #define BL_PROFILE_T_S(fname, T)
 
-#define BL_PROFILE_VAR(fname, vname)      amrex::TinyProfiler tiny_profiler__##vname((fname));
-#define BL_PROFILE_VAR_NS(fname, vname)   amrex::TinyProfiler tiny_profiler__##vname(fname, false);
-#define BL_PROFILE_VAR_START(vname)       tiny_profiler__##vname.start();
-#define BL_PROFILE_VAR_STOP(vname)        tiny_profiler__##vname.stop();
+#define BL_PROFILE_VAR(fname, vname)                      amrex::TinyProfiler tiny_profiler__##vname((fname));                                                                                                          
+#define BL_PROFILE_VAR_NS(fname, vname)                   amrex::TinyProfiler tiny_profiler__##vname(fname, false, false);                                                                                              
+#define BL_PROFILE_VAR_START(vname)                       tiny_profiler__##vname.start();                                                                                                                               
+#define BL_PROFILE_VAR_STOP(vname)                        tiny_profiler__##vname.stop(); 
+#ifdef AMREX_USE_CUPTI
+  #include <AMReX_CuptiTrace.H>
+  #define BL_PROFILE_VAR_NS_CUPTI(fname, vname)             amrex::TinyProfiler tiny_profiler__##vname(fname, false, true);
+  #define BL_PROFILE_VAR_START_CUPTI(vname)                 tiny_profiler__##vname.start();
+  #define BL_PROFILE_VAR_STOP_CUPTI(vname)                  tiny_profiler__##vname.stop();
+  #define BL_PROFILE_VAR_STOP_CUPTI_ID(vname, uintID)       tiny_profiler__##vname.stop(uintID);
```

Comment:
The standard says `#` has to be the first character of a line.

Reply:
Done.